aiven/resource_transit_gateway_vpc_attachement.go

```diff
@@ -81,7 +81,7 @@ func resourceTransitGatewayVPCAttachment() *schema.Resource {
 func resourceTransitGatewayVPCAttachmentUpdate(d *schema.ResourceData, m interface{}) error {
 	client := m.(*aiven.Client)
 
-	cidrs := d.Get("user_peer_network_cidrs").([]string)
+	cidrs := flattenToString(d.Get("user_peer_network_cidrs").([]interface{}))
```

Comment:
Fwiw this function name is quite confusing. I'd expect it to flatten an array into string instead of mapping individual array elements into strings.